### DISCUSSION TITLE
Full date

### DISCUSSION THREAD

**Discussion by tapochek666** [JanDeDobbeleer/oh-my-posh#5708] [General]
Hi. How can I display the full date in Russian in the oh my posh theme?For now it's like this
{
          "background": "#0000FF",
          "foreground": "#fcfcfc",
          "properties": {
            "time_format": "Monday 15:04"
          },
          "style": "diamond",
          "template": " \uf017  {{ .CurrentDate | date .Format }} ",
          "trailing_diamond": "\ue0b4",
          "type": "time"
        },
**Sorry if I asked the question in the wrong place**

**Comment by JanDeDobbeleer** [comment]
@tapochek666 have a look at the [predefined formats](https://ohmyposh.dev/docs/segments/system/time#predefined-formats), but translation isn't something that's supported in oh-my-posh.

**Reply by tapochek666** [reply]
If the translation is not supported, then why do I need a link!?

**Reply by JanDeDobbeleer** [reply]
@tapochek666 you asked about the full date as well so I assumed that was also a question?

**Reply by tapochek666** [reply]
I asked about the full date, I know how to display it in English... The essence of the question was how to display it in Russian. You answered that the date is not translated.

**Reply by JanDeDobbeleer** [reply]
@tapochek666 not supported indeed. Also won't ever be supported.

**Reply by tapochek666** [reply]
Sanctions?

**Reply by JanDeDobbeleer** [reply]
@tapochek666 the language is irrelevant, I'm not adding translation capabilities regardless 😄

**Reply by tapochek666** [reply]
Well, "thank you" from all Russian-speaking citizens!

**Reply by kostyasimanov** [reply]
> Well, "thank you" from all Russian-speaking citizens!

I'm a Russian-speaking citizen, and this case doesn't concern me, and I'm really say thanks to Jan, not "thank you" in sarcastic tone like you. 
If you were smarter and patiently, you'd try googling the time format in oh-my-posh and find this issue from a Spanish-speaking citizen, published in November 2023 with work-around:
https://github.com/JanDeDobbeleer/oh-my-posh/issues/4469

Just don't look for sanctions in every answer you don't like.

**Comment by tapochek666** [comment]
@kostyasimanov ну и говори спасибо дальше!Я эту проблему решил позже с помощью чата gpt.Поигрался и удалил сее ***. Нет,надо было спустя год нарисоваться со своим комментарием....
